src/main/java/com/epam/lab/business/ProfilePageBo.java

```diff
@@ -135,9 +135,10 @@ public String getFullClassname(String firstPart, String secondPart) {
     }
     
     @Step("PageElementBO Step:  get and click All Contacts Button...")
-	  public void getAndClickAllContactsButton() {
-		profilePagePo.getAndClickAllContactsButton();
-	  }
+    public void getAndClickAllContactsButton() {		
+	JavascriptExecutor js = (JavascriptExecutor) DriverManager.getDriver();
```

Comment:
you should use page object here, not driver

Reply:
Sorry - I copypasted this code, not thinking about Liskov Principle

Reply:
I am working on this problem,but soo slow...